merlion/utils/misc.py

```diff
@@ -30,6 +32,83 @@ def __new__(mcls, classname, bases, cls_dict):
         return cls
 
 
+class ModelConfigMeta(type):
+    """
+    Metaclass used to ensure that the function signatures for model `Config` initializers contain all
+    relevant parameters, including those specified in the superclass. Also update docstrings accordingly.
```

Comment:
I feel like an example here would be helpful

Reply:
Done.